exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,9 @@
 const name = " Daniel  ";
 
-console.log(message);
+let firstName = "Ahmad";
```

Comment:
Your name variable should have some extra spaces here, so when you use trim, It has something to remove.